### PR TITLE
Documentation for R4 AllergyIntolerance Update

### PR DESCRIPTION
**Pull request by JFisk42**

### CODE REVIEW

File: content/millennium/r4/general-clinical/allergy-intolerance.md [hunk 1]
```diff
@@ -172,3 +167,68 @@ The `ETag` response header indicates the current `If-Match` version to use on a
...
+    Last-Modified: Thu, 05 Dec 2019 17:21:08 GMT
+    Server-Response-Time: 743.187771
+    Status: 200 OK
+    X-Request-Id: a53f6469ff29031e9197b40f526e9ca6
+    X-Runtime: 0.743187
+    Vary: Origin
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
```
Comment (praneethk2094) [inline content/millennium/r4/general-clinical/allergy-intolerance.md]: Do we want to do the disclaimer here
```
<%= disclaimer %>
```

File: lib/resources/r4/allergy_intolerance.yaml [hunk 2]
```diff
@@ -298,6 +339,43 @@ fields:
...
+  type: Annotation
+  description: Additional narrative about the allergy not captured in other fields.
+  note: Updates do not allow an author to be specified. One will be infered from the authorization token. If an id is not
+        provided with the note it will be assumed to be a new note.
+  action: update
+  url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.note
+  example: |
+    {
+      "note": [
+        {
```
Comment (praneethk2094) [inline lib/resources/r4/allergy_intolerance.yaml]: Could we update this field to better reflect notes with ID's and notes without.

File: lib/resources/example_json/r4_examples_allergy_intolerance.rb [hunk 3]
```diff
@@ -211,5 +219,53 @@ module Resources
...
+          "display": "SMART, WILMA"
+      },
+      "recordedDate": "2019-02-08T00:00:00Z",
+      "recorder": {
+          "reference": "Practitioner/2042017",
+          "display": "Generated Domain User for 0"
+      },
+      "note": [
+        {
+          "text": "Updated with additional details provided by patient."
```
Comment (praneethk2094) [inline lib/resources/example_json/r4_examples_allergy_intolerance.rb]: Could we update this field to better reflect notes with ID's and notes without?

**Comment by praneethk2094** [comment]
Local Verification of updates
Screen Shot 2019-12-05 at 5 01 54 PM
Screen Shot 2019-12-05 at 4 50 58 PM
Screen Shot 2019-12-05 at 4 40 45 PM

**Comment by sbanda51** [comment]
+1 after addressing the comments

File: content/millennium/r4/general-clinical/allergy-intolerance.md (hunk 1 above)
Comment (JFisk42) [reply content/millennium/r4/general-clinical/allergy-intolerance.md]: Yeah we do, added: https://github.com/cerner/fhir.cerner.com/pull/327/commits/31f33cdc9cac89a6e5fc99c1a6d77e2683f8d674

File: lib/resources/example_json/r4_examples_allergy_intolerance.rb (hunk 3 above)
Comment (JFisk42) [reply lib/resources/example_json/r4_examples_allergy_intolerance.rb]: Added an existing note field, do you think the note text is too on the nose? https://github.com/cerner/fhir.cerner.com/pull/327/commits/31f33cdc9cac89a6e5fc99c1a6d77e2683f8d674

Comment (praneethk2094) [reply lib/resources/example_json/r4_examples_allergy_intolerance.rb]: To me it's perfectly where it should be. But, i am wondering about out strategy on if we want to keep our example body's understandable or medically accurate. 

Thoughts?

Comment (JFisk42) [reply lib/resources/example_json/r4_examples_allergy_intolerance.rb]: After discussion offline we decided to go with more medically accurate note descriptions and then add extra documentation detailing how note works. https://github.com/cerner/fhir.cerner.com/pull/327/commits/57930130970b544cc9a59acfc45e6ad0583c87fb @praneethk2094  please review this commit.

**Comment by praneethk2094** [comment]
Implementation notes added. 
Screen Shot 2019-12-06 at 1 20 44 PM

**Comment by praneethk2094** [comment]
Implementation notes added. 
Screen Shot 2019-12-06 at 1 20 44 PM

**Comment by nishanthnamana** [comment]
Verification on fhir.cerner.com

Allergy_update_1

Allergy_update_2

**Pull request merged**